src/components/ButtonList-review.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export default function ButtonList() {
+  return (
+    <div className="button-panel">
+      <div>
+        <b>General</b>
+      </div>
+
+      <nav className="buttons">
+        <button>
+          <Link to="/value">Full Time</Link>
```

Comment:
would be good if the path names reflect the data they hold, for example instead of using 'value' you could come up with a path name that indicates that page holds full time data